rdr_service/dao/bq_questionnaire_dao.py

```diff
@@ -25,6 +29,59 @@ def make_bqrecord(self, p_id, module_id, latest=False, convert_to_enum=False):
         :param convert_to_enum: If schema field description includes Enum class info, convert value to Enum.
         :return: BQTable object, List of BQRecord objects
         """
+
+        # Query to get all the question codes across all questionnaire versions of the module.
+        _question_code_sql = """
+            -- select c.code_id, c.value
+            select c.value
+            from code c
+            inner join (
+                select distinct qq.code_id
+                from questionnaire_question qq where qq.questionnaire_id in (
+                    select qc.questionnaire_id from questionnaire_concept qc
+                            where qc.code_id = (
+                                select code_id from code c2 where c2.value = :module_id )
+                )
+            ) qq2 on qq2.code_id = c.code_id
+            order by c.code_id;
+        """
+
+        # Query to get a list of questionnaire_response_id values for this participant and module
+        _participant_module_responses_sql = """
+            select qr.questionnaire_id, qr.questionnaire_response_id, qr.created, qr.authored, qr.language,
+                   qr.participant_id
+            from questionnaire_response qr
+            where qr.participant_id = :p_id and questionnaire_id IN (
+                select q.questionnaire_id from questionnaire q
+                inner join questionnaire_history qh on q.version = qh.version
+                       and qh.questionnaire_id = q.questionnaire_id
+                inner join questionnaire_concept qc on qc.questionnaire_id = q.questionnaire_id
```

Comment:
It might not affect anything, but to be safe `questionnaire_concept` should probably be joined on the version too.

Reply:
Looking.

Reply:
Fixed in most recent changes